addons/website_event/controllers/main.py

```diff
@@ -389,6 +389,17 @@ def registration_confirm(self, event, **post):
         registrations = self._process_attendees_form(event, post)
         attendees_sudo = self._create_attendees_from_registration_post(event, registrations)
 
+        return request.redirect('/event/%s/registration/success?registration_ids=%s' % (event.id, ",".join([str(id) for id in attendees_sudo.ids])))
+
+    @http.route(['/event/<model("event.event"):event>/registration/success'], type='http', auth="public", methods=['GET'], website=True, sitemap=False)
+    def event_registration_success(self, event, registration_ids):
+        # fetch the related registrations, make sure they belong to the correct visitor / event pair
+        visitor = request.env['website.visitor']._get_visitor_from_request()
```

Comment:
People may try to call it from outside of expected flow. I would probably do something if we find no visitor.